src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+name contains the domain name of the issuing organization, a colon and a type
+name that is at least one character long, for instance "example.com:foo".
+
+```js
+function readExternalTypeRecord(externalTypeRecord) {
+  for (const record of externalTypeRecord.toRecords()) {
+    if (record.recordType == "text") {
+      const decoder = new TextDecoder(record.encoding);
+      console.log(`Text: ${textDecoder.decode(record.data)} (${record.lang})`);
+    } else if (record.recordType == "url") {
+      const decoder = new TextDecoder();
+      console.log(`URL: ${decoder.decode(record.data)}`);
+    } else {
+      // TODO: Handle other type of records.
+    }
+  }
+}
+```
+
+To write an external type record, pass an NDEF message dictionary to the
+NDEFWriter `write()` method. The external type record contained in the NDEF
+message is defined as an object with a `recordType` key set to the name of the
+external type and a `data` key set to an object that represents (once again) an
+NDEF message contained in the external type record.  Note that the `data` key
+can also be either an ArrayBuffer or provides a view on to an ArrayBuffer (.e.g.
+Uint8Array, DataView).
+
+```js
+const externalTypeRecord = {
+  recordType: "example.game:a",
+  data: {
+    records: [
+      {
+        recordType: "url",
+        data: "https://example.game/42"
+      },
+      {
+        recordType: "text",
+        data: "Game context given here"
+      },
+      {
+        recordType: "mime",
+        mediaType: "image/png",
+        data: await (await fetch("image.png")).arrayBuffer()
+      }
+    ]
+  }
+};
+
+const writer = new NDEFWriter();
+writer.write({ records: [externalTypeRecord] });
+```
+
+### Read and write an empty record
+
+An empty record has no payload.
+
+To write an empty record, pass an NDEF message dictionary to the NDEFWriter
+`write()` method. The empty record contained in the NDEF message is defined as
+an object with a `recordType` key set to `"empty"`.
+
+```js
+const emptyRecord = {
+  recordType: "empty"
+};
+
+const writer = new NDEFWriter();
+await writer.write({ records: [emptyRecord] });
+```
+
+## Dev Tips
+
+Here's a list of things I wish I had known when I started playing with Web NFC:
+
+- Android natively handles NFC tags before Web NFC is operational.
+- An NFC icon can be found on [material.io].
+- Use NDEF record `id` to simply identifying a record when needed.
+- An unformatted NFC tag that supports NDEF contains an NDEF message with an
+  empty record only.
+- Writing an [android application record] is easy, as shown below.
+
+```js
+const encoder = new TextEncoder();
+const aarRecord = {
+  recordType: "android.com:pkg",
+  data: encoder.encode("com.example.myapp")
+};
+
+const writer = new NDEFWriter();
+await writer.write({ records: [aarRecord] });
+```
+
+## Demos {: #demos }
+
+Give a try to the [official sample] and check out some cool Web NFC demos:
```

Comment:
give a try to, sounds weird... Try out the...

Reply:
Sure.